build-scripts/build-csdk.js

```diff
@@ -57,7 +59,17 @@ var archMap = {
 	"linux": {
 			"x64": "x86_64",
 			"arm": "arm"
-			}
+			},
```

Comment:
Please reduce the indentation such that the brace aligns with the double quote before the word `"linux"` above.